main.nf

```diff
@@ -163,7 +163,7 @@ process MapReads {
   script:
   readGroup = "@RG\\tID:$idRun\\tSM:$idSample\\tLB:$idSample\\tPL:illumina"
   // adjust mismatch penalty for tumor samples
-  extra = status == 1 ? "-B 3 " : ""
+  extra = status == "1" ? "-B 3 " : ""
```

Comment:
Wouldn't it be slightly better to actually turn status into an int? And if you want it to be a string, why not use 'tumor' and 'normal'?